packages/apps/job-launcher/server/src/modules/payment/payment.service.ts

```diff
@@ -229,13 +228,13 @@ export class PaymentService {
       throw new BadRequestException(ErrorPayment.TransactionAlreadyExists);
     }
 
-    const rate = await getRate(Currency.USD, TokenId.HMT);
+    const rate = await getRate(Currency.USD, tokenId);
 
     await this.paymentRepository.create({
       userId,
       source: PaymentSource.CRYPTO,
       type: PaymentType.DEPOSIT,
-      amount,
+      amount: mul(amount, rate),
```

Comment:
In createCryptoPayment we were saving in paymentEntity.amount the amount in tokens and this field only should save USD.
- When recording new payments in the database, it's vital to retain the original account value prior to any currency conversions. This ensures the integrity and precision of the recorded data.

Reply:
related to this https://github.com/humanprotocol/human-protocol/pull/815#discussion_r1302525476